tests/Craftwork/IdObfuscator/IdObfuscatorTest.php

```diff
@@ -0,0 +1,57 @@
+<?php
+namespace Craftwork\IdObfuscator;
+
+use PHPUnit\Framework\TestCase;
+
+class IdObfuscatorTest extends TestCase
+{
+    /**
+     * @var IdObfuscator
+     */
+    private $idObfuscator;
+
+    public function testEncodeIdWithoutSalt()
+    {
+        $this->idObfuscator = $this->createIdObfuscator('');
```

Comment:
I don't see the added value of having $idObfuscator as a property, and also the createIdObfuscator method.

Just doing `$idObfuscator = new IdObfuscator('');` is clearer and more concise.